src/test/java/seedu/address/logic/commands/ImportContactsCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import org.junit.Test;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ImportContactsCommandTest {
+    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "ImportContactsTest");
```

Comment:
Can consider using @Rule to create a temporary file for this test that will auto-delete when test finishes running